meta/main.yml

```diff
@@ -0,0 +1,28 @@
+galaxy_info:
+    author: LSST SQuaRE Team
+    description: Docker Latest Offical Release
+    company: LSST SQuaRE
+    issue_tracker_url: https://github.com/lsst-sqre/ansible-logstash-packages/issues
+    license: MIT
+    min_ansible_version: 2.1
+    platforms:
+    - name: Debian
+      versions:
+        - stretch
```

Comment:
Can travis test more of these specific Linux versions? I'll see which versions are available to me at NCSA and test what I can. I think the priority is CentOS7 and Xenial though.